inference-engine/tests/functional/plugin/gna/shared_tests_instances/single_layer_tests/convolution.cpp

```diff
@@ -188,7 +208,7 @@ INSTANTIATE_TEST_CASE_P(smoke_Convolution2D_AutoPadValid_MapTo1d, ConvolutionLay
                         ConvolutionLayerTest::getTestCaseName);
 
 // TODO: Enable for GNA 2.1 library
-INSTANTIATE_TEST_CASE_P(DISABLED_smoke_Convolution2D_Kernels2D, ConvolutionLayerTest,
+INSTANTIATE_TEST_CASE_P(smoke_Convolution2D_Kernels2D, GnaConvolutionLayerTest,
```

Comment:
Apply this helper and enable tests:
INSTANTIATE_TEST_CASE_P(DISABLED_smoke_ConvolutionReluSequenceTest, ConvolutionReluSequenceTest,
in:
https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/tests/functional/plugin/gna/shared_tests_instances/subgraph_tests/convolution_relu_sequence.cpp

Reply:
Done.